javascripts/discourse/api-initializers/discourse-fakebook.gjs

```diff
@@ -0,0 +1,51 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+import dIcon from "discourse-common/helpers/d-icon";
+import CustomSidebar from "../components/custom-sidebar";
+import TopicExcerpt from "../components/topic-excerpt";
+import TopicLikes from "../components/topic-likes";
+import TopicOp from "../components/topic-op";
+import TopicTags from "../components/topic-tags";
+import TopicThumbnail from "../components/topic-thumbnail";
+
+export default {
+  name: "discourse-fakebook",
+
+  initialize() {
```

Comment:
This is in the `api-initializers` directory, but not actually implemented using `apiInitializer()`

We should either move it to the normal `initializers` directory, or switch it to use `apiInitializer`

Reply:
Done